docs/src/osdk/guide/intel-tdx-guide.md

```diff
@@ -0,0 +1,93 @@
+# Running an OS in Intel TDX env
+
+OSDK supports running your OS in an [Intel TDX](https://www.intel.com/content/www/us/en/developer/tools/trust-domain-extensions/overview.html) environment conveniently. Intel TDX can provide a more secure environment for your OS.
+
+## Intel TDX environment preparation
```

Comment:
"Intel TDX environment preparation," the header should be more grammatically consistent with other headers. 

Suggested change:
"Prepare the Intel TDX Environment"